Modules/FindCapistrano.cmake

```diff
@@ -105,5 +108,7 @@ IF(CAPISTRANO_FOUND)
     SET(CAPISTRANO_LIBRARY_BMODE_NAME "BmodeUSB${Capistrano_LIBRARY_FILE_NAME_SUFFIX}")
   ELSEIF(Capistrano_SDK_VERSION STREQUAL "cSDK2016" OR Capistrano_SDK_VERSION STREQUAL "cSDK2013")
     SET(CAPISTRANO_LIBRARY_BMODE_NAME "Bmode${Capistrano_LIBRARY_FILE_NAME_SUFFIX}")
+  ELSE()
+    MESSAGE(FATAL_ERROR "Unknown version of Capistrano SDK")

```

Comment:
Agreed, but as a person who has done `-DPLUSBUILD_EXTERNAL_VTK_VERSION:STRING="9.0.0"`, having a catch is very useful (should have been `"v9.0.0.0"`).

Thanks!